### PR TITLE
[wip] rework details on setuptools_scm/setuptools

### PR DESCRIPTION
**Pull request by RonnyPfannschmidt**

### CODE REVIEW

File: topics/version.md [hunk 1]
```diff
@@ -12,17 +12,21 @@ Specify a package version that can be used by package managers at install time,
 
 ### Setuptools (vanilla)
 
-Setuptools by default does not make any efforts in this regard. The user is expected to pass a string to `setup()`, and that’s it. You need to either duplicate the value in code if you want to read the value, or use `pkg_resources` to read out the installed metadata.
+Setuptools only concerns itself with metadata, the version passed in for the `setup()` is only transfered into the package metadata.
+Making it availiable as a variable is expected to be solved by the maintainer.
+
+Its generally suggested to query the package metadata for the version to ensure a single source of truth.
```
Comment (brettcannon) [inline topics/version.md:18]:
```suggestion
It's generally suggested to query the package metadata for the version to ensure a single source of truth.
```

**Review by brettcannon — CHANGES_REQUESTED** [review]
One very minor tweak and then I think this is ready to be merged!

**Comment by uranusjr** [comment]
@RonnyPfannschmidt Title says WIP but this is still better than master, so I’m just going to merge this. We can always modify it further.

**Comment by uranusjr** [comment]
(I manually added Brett’s change request in c5b1ea714)

**Pull request merged**